frontend/public/components/operator-lifecycle-manager/clusterserviceversion.tsx

```diff
@@ -264,13 +270,45 @@ export const ClusterServiceVersionDetails: React.SFC<ClusterServiceVersionDetail
       </div>
     </div>
     <div className="co-m-pane__body">
-      {/* FIXME(alecmerdler): Need to modify CSV conditions to follow standard conventions */}
       <SectionHeading text="Conditions" />
       <Conditions conditions={_.get(status, 'conditions', []).map(c => ({...c, type: c.phase, status: 'True'}))} />
     </div>
   </React.Fragment>;
 };
 
+export const CSVSubscription: React.FC<CSVSubscriptionProps> = (props) => {
+  type SubscriptionProps = {
+    subscription: {loaded: boolean, data?: SubscriptionKind[]};
+    packageManifest: {loaded: boolean, data?: PackageManifestKind[]};
+    loaded: boolean;
+  };
+
+  const Subscription: React.FC<SubscriptionProps> = (subscriptionProps) => {
+    const subscription = subscriptionProps.subscription.data.find(sub => _.get(sub.status, 'installedCSV') === props.installedCSV.metadata.name);
+
+    return subscriptionProps.loaded
+      ? <SubscriptionDetails
+        obj={subscription}
+        installedCSV={props.installedCSV}
+        pkg={subscriptionProps.packageManifest.data.find(pkg => pkg.status.packageName === subscription.spec.name)} />
+      : <LoadingBox />;
```

Comment:
Do we need to handle load errors?

Reply:
Added a `<StatusBox>` to handle load errors.